app/content/tiles/jadeCityLight.ts

```diff
@@ -84,32 +84,33 @@ const jadeCityLightWalls: TileSource = {
 }
 
 
+
 const jadeCityLightSlopedWalls: TileSource = {
     w: 16, h: 16,
     source: requireFrame('gfx/tiles/jadeCityLight.png', {x: 272, y: 112, w: 128, h: 112}),
     behaviors: {
-        'all': southernWallBehavior,
-        '0x0': { defaultLayer: 'field', solidMap: BITMAP_BOTTOM_LEFT_CORN_10 },
-        '0x1': { defaultLayer: 'field', solidMap: BITMAP_TOP_RIGHT_MISS_CORN_6 },
+        'all': {solid: true, defaultLayer: 'field', isSouthernWall: true, linkedOffset: 159},
```

Comment:
Use object spreading here and in other places with similar code: `{...southernWallBehavior, linkedOffset: 159}`